x/checkers/keeper/msg_server_play_move.go

```diff
@@ -52,6 +53,17 @@ func (k msgServer) PlayMove(goCtx context.Context, msg *types.MsgPlayMove) (*typ
 	// Save for the next play move
 	k.Keeper.SetStoredGame(ctx, *fullGame.ToStoredGame())
 
+	// What to emit
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(types.PlayMoveEventKey,
+			sdk.NewAttribute(types.PlayMoveEventCreator, msg.Creator),
+			sdk.NewAttribute(types.PlayMoveEventIdValue, msg.IdValue),
+			sdk.NewAttribute(types.PlayMoveEventCapturedX, strconv.FormatInt(int64(captured.X), 10)),
+			sdk.NewAttribute(types.PlayMoveEventCapturedY, strconv.FormatInt(int64(captured.Y), 10)),
+			sdk.NewAttribute(types.PlayMoveEventWinner, fullGame.Game.Winner().Color),
+		),
+	)
```

Comment:
```suggestion
	ctx.EventManager().EmitEvent(
		sdk.NewEvent(sdk.EventTypeMessage,
		sdk.NewAttribute(sdk.AttributeKeyModule, types.AttributeValueCategory),
		sdk.NewAttribute(sdk.AttributeKeyAction, types.PlayMoveEventKey),
			sdk.NewAttribute(types.PlayMoveEventCreator, msg.Creator),
			sdk.NewAttribute(types.PlayMoveEventIdValue, msg.IdValue),
			sdk.NewAttribute(types.PlayMoveEventCapturedX, strconv.FormatInt(int64(captured.X), 10)),
			sdk.NewAttribute(types.PlayMoveEventCapturedY, strconv.FormatInt(int64(captured.Y), 10)),
			sdk.NewAttribute(types.PlayMoveEventWinner, fullGame.Game.Winner().Color),
		),
	)
```

this is more in tune with sdk conventions.

Reply:
What do you have in mind for the content of `types.AttributeValueCategory`?

Seeing what is declared after `sdk.EventTypeMessage` I wonder if I should also add:

```go
sdk.NewAttribute(sdk.AttributeKeyModule, "checkers")
```

Reply:
you could use the module name. not sure if you have it defined in the types folder. This is easier than using a string.

Reply:
And what about `types.AttributeValueCategory`?

Reply:
in sdk based modules AttributeValueCategory is an alias of moduleName